exporter/azure_monitor/azure_monitor.go

```diff
@@ -0,0 +1,46 @@
+package azure_monitor
+
+import (
+	"fmt"
+	"errors"
+	"go.opencensus.io/exporter/azure_monitor/common"
+	"go.opencensus.io/trace"
+)
+
+type AzureTraceExporter struct {
+	projectID          string
+	InstrumentationKey string
+	options            common.Options
+}
+
+func NewAzureTraceExporter(o common.Options) (*AzureTraceExporter, error) {
+	if o.InstrumentationKey == "" {
+		return nil, errors.New("missing Instrumentation Key for Azure Exporter")
+	}
+	e := &AzureTraceExporter {
+		projectID:          "abcdefghijk",
+		InstrumentationKey: o.InstrumentationKey,
+		options:            o,
+	}
+	return e, nil
+}
+
+var _ trace.Exporter = (*AzureTraceExporter)(nil)
+
+// Export SpanData to Azure Monitor
+// The () before the function name means it is a function of AzureTraceExporter
+func (e *AzureTraceExporter) ExportSpan(sd *trace.SpanData) {

```

Comment:
Having comments for your functions is good practice. After you implement this, you should include in your comments what your function does, the parameters that they take in and the data types. You can remove the second line comment in the future because that is more language specific information.

Reply:
Okay, I will add more comments to my functions. Yes, parameters and data type are a good idea as well. I plan to remove the golang specific line after this PR. It was confusing for me when I first learned it so I mentioned it.